Index: sites/all/modules/contrib/user_registrationpassword/user_registrationpassword.pages.inc
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- sites/all/modules/contrib/user_registrationpassword/user_registrationpassword.pages.inc	(revision 4ac30481e2586cc9fe4a6a198f0d730ec8d56a0f)
+++ sites/all/modules/contrib/user_registrationpassword/user_registrationpassword.pages.inc	(revision )
@@ -64,7 +64,8 @@
     else {
       // Deny access, no more clues.
       // Everything will be in the watchdog's URL for the administrator to check.
-      drupal_access_denied();
+      drupal_set_message(t('We are sorry, but we can not confirm your registration. Maybe your confirmation link is no longer valid. Please <a href="/contact">contact us</a>, we will try to find a solution!'), 'error');
+      drupal_goto();
     }
   }
 }
